module/VuFind/tests/unit-tests/src/VuFindTest/ILS/Driver/DAIATest.php

```diff
@@ -49,93 +49,97 @@ class DAIATest extends \VuFindTest\Unit\ILSDriverTestCase
-                'storageid' => '',
-                'storagehref' => '',
-                'item_notes' => [],
-                'services' => [],
-                'is_holdable' => false,
-                'addLink' => false,
-                'holdtype' => null,
-                'addStorageRetrievalRequestLink' => false,
-                'customData' => [],
-                'limitation_types' => [],
-                'doc_id' => 'http://uri.gbv.de/document/opac-de-000:ppn:027586081'
-            ],
+        'holdings' => [
+            0 =>
+                [
+                    'status' =>    '',
+                    'availability' => true,
+                    'duedate' => null,
+                    'requests_placed' => '',
+                    'id' => "027586081",
+                    'item_id' => "http://uri.gbv.de/document/opac-de-000:epn:711134758",
+                    'ilslink' => "http://opac.example-library.edu/loan/REQ?EPN=711134758",
+                    'number' => 1,
+                    'barcode' => "1",
+                    'reserve' => "N",
+                    'callnumber' => "ABC 12",
+                    'location' => 'Example Library for DAIA Tests',
+                    'locationid' => 'http://uri.gbv.de/organization/isil/DE-000',
+                    'locationhref' => 'http://www.example-library.edu',
+                    'storage' => 'Abteilung III',
+                    'storageid' => '',
+                    'storagehref' => '',
+                    'item_notes' => [],
+                    'services' => ['presentation', 'loan'],
+                    'is_holdable' => false,
+                    'addLink' => false,
+                    'holdtype' => null,
+                    'addStorageRetrievalRequestLink' => true,
+                    'customData' => [],
+                    'limitation_types' => [],
+                    'doc_id' => 'http://uri.gbv.de/document/opac-de-000:ppn:027586081'
+                ],
+            1 =>
+                [
+                    'status' => '',
+                    'availability' => true,
+                    'duedate' => null,
+                    'requests_placed' => '',
+                    'id' => "027586081",
+                    'item_id' => "http://uri.gbv.de/document/opac-de-000:epn:711134766",
+                    'ilslink' => "http://opac.example-library.edu/DB=1/PPNSET?PPN=027586081",
+                    'number' => 2,
+                    'barcode' => "1",
+                    'reserve' => "N",
+                    'callnumber' => "DEF 34",
+                    'location' => 'Example Library for DAIA Tests',
+                    'locationid' => 'http://uri.gbv.de/organization/isil/DE-000',
+                    'locationhref' => 'http://www.example-library.edu',
+                    'storage' => 'Abteilung III',
+                    'storageid' => '',
+                    'storagehref' => '',
+                    'item_notes' => ['mit Zustimmung'],
```

Comment:
I just notice, that this might be a limitation_type rather than an item_note...

Reply:
@olli-gold: a limitation_type contains the id of a limitation, which is in any case should be a purl. Only the contents of limitations are mapped to item_notes. In fact I've never seen a limitation id in real DAIA; limitations seem to be announced only in a free-text style.

Reply:
I see the problem, that we do not get an ID from our DAIA provider, but the specification recommends to set IDs (cf. https://gbv.github.io/daia/daia.html#limitations). Maybe we should ask for IDs at our DAIA provider, but it will certainly take much time until we really have them. In the meantime we could choose to either add IDs in the driver (at least for some known free text limitations) or to keep the status of showing the limitations as item_notes...

Reply:
But maybe we should open a different PR for that development, as it really does not need to be addressed here.